test/dummy/app/components/dummy/mailer/header_component.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Dummy::Mailer::HeaderComponent < ApplicationComponent
+  def initialize(site:, logo: false)
```

Comment:
bez `logo` se to nikde nepouziva, muzes vyhodit

Reply:
Pridaval jsem to na zaklade poznamky "Optional logo or H1 headline" ve Figme
![image](https://github.com/sinfin/folio/assets/83029101/5d3eec85-5aee-4e4f-b661-7092322db7cb)